src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2142,7 +2145,7 @@ public Field getField(String name) throws NoSuchFieldException {
      * overriding method as it would have a more specific return type.
      *
      * @param name the name of the method
-     * @param parameterTypes the list of parameters
+     * @param parameterTypes the list of parameters, may be or contain {@code null}
```

Comment:
The method description already specifies that if parameterTypes is null then it works like an empty array, so "may be null" is okay here.

getMethod, getConstructor, ... throw NoSuchMethodException is the parameterTypes has a null element. Maybe it would be better to expand the description for `@throws NoSuchMethodException`? That would avoid suggesting in the parameter description that a null element is allowed.